scripts/release/issue-list.sh

```diff
@@ -17,9 +17,10 @@
 programname=$0
 function usage()
 {
-    echo "usage: $programname -r repository -m milestone"
+    echo "usage: $programname -r repository -m milestone -f format"
     echo "  -r  repository name"
     echo "  -m  milestone id"
+    echo "  -f  output format ('human' or 'jsonstring')"
```

Comment:
@hferentschik , Tried above. This is how it is currently looking 

https://github.com/minishift-bot/minishift/releases/v1.2.4

Noticed that whole content is wrapped as single string. Also no `\n` character.

Any idea how to manipulate it through `jq`. Honestly, never used `jq` much before this. Just trying out myself.
![screenshot from 2017-07-06 16-40-22](https://user-images.githubusercontent.com/1132451/27908520-d838cefc-6269-11e7-89ea-81d4d28514e6.png)